### ISSUE TITLE
Schema:update allways shows diff with Carbondatetime customTypeDiff

### ISSUE THREAD

**Issue by NaGeL182** [doctrine/dbal#3265]
### Bug Report

|    Q        |   A
|------------ | ------
| BC Break    |no
| Version     | 2.8.0

#### Summary

Similar to issue #2596 I have a Custom type with `beberlei/DoctrineExtensions` a `CarbonDateTimeType` and i have several field with that type.  The Migration file gets created correctly:

```sql
...
created_at DATETIME NOT NULL COMMENT '(DC2Type:carbondatetime)', updated_at DATETIME DEFAULT NULL COMMENT '(DC2Type:carbondatetime)',
...
```
but when i give out the command `bin/console doctrine:schema:update --dump-sql` it gives outthe following diff each and every time:

```sql
...
ALTER TABLE `table` CHANGE created_at created_at DATETIME DEFAULT CURRENT_TIMESTAMP NOT NULL COMMENT '(DC2Type:carbondatetime)', CHANGE updated_at updated_at DATETIME DEFAULT NULL COMMENT '(DC2Type:carbondatetime)';

...
```

#### Current behavior

Notices diffrences where there is none! 

#### How to reproduce

create an Entity with https://github.com/beberlei/DoctrineExtensions `carbondatetime` type field
update the database
check update SQL

#### Expected behaviour

It doesnt sees the customType fields as changed.

**Comment by NaGeL182** [comment]
hey, this seems to be fixed in `doctrine/orm` `v2.6.2`!
I don't get those diffs anymore! dunno what fixed it tho because the dbal is still at `2.8.0`...

**Comment by Ocramius** [comment]
Closing here then :+1:
